sandbox/host/src/sandbox/wasmer_backend.rs

```diff
@@ -361,13 +401,13 @@ fn dispatch_function(
 
             let serialized_result_val = serialized_result_val?;
 
-            let deserialized_result = std::result::Result::<ReturnValue, HostError>::decode(
+            let deserialized_result = std::result::Result::<WasmReturnValue, HostError>::decode(
```

Comment:
Won't it break backward compatibility for older nodes?

Reply:
Good catch, thanks! :+1: 

Actually it would not break older nodes it just would bring to the case that new nodes wouldn't be able to synchronize old chain state. So we have to introduce new method version and support also the old one.